tests/lisa/test_energy_model.py

```diff
@@ -0,0 +1,253 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2016, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from collections import OrderedDict
+import unittest
+from unittest import TestCase
+
+from energy_model import (EnergyModel, ActiveState, EnergyModelCapacityError,
+                          EnergyModelNode, EnergyModelRoot, PowerDomain)
+
+"""
+A very basic test for the EnergyModel class.
+
+Just creates a simple instance of EnergyModel describing a hypothetical
+big.LITTLE system and smoke-tests the methods and properties.
+"""
+
+little_cluster_active_states = OrderedDict([
+    (1000, ActiveState(power=10)),
+    (2000, ActiveState(power=20)),
+])
+
+little_cluster_idle_states = OrderedDict([
+    ('WFI',              5),
+    ('cpu-sleep-0',      5),
+    ('cluster-sleep-0',  1),
+])
```

Comment:
Sounds good.

I did also implement a [method to print various observations about the energy model](https://github.com/bjackman/lisa/blob/energy-model-rfc-v2-evaluation/libs/utils/energy_model.py#L793) but didn't include it here.